src/components/ec-text-field/ec-text-field.story.js

```diff
@@ -0,0 +1,66 @@
+import { storiesOf } from '@storybook/vue';
+import { text, select } from '@storybook/addon-knobs';
+import EcTextField from './ec-text-field.vue';
+
+const stories = storiesOf('TextField', module);
```

Comment:
We do have 
"Dropdown Seach"
"Privacy Policy"
"Inline Actions"

Could you please match the text to the pattern? It would be nice to fix the "Donut chart" too.